README.md

```diff
@@ -1,3 +1,3 @@
 # Google Student Training in Engineering Program
 
-This repo contains Andru Stefanescu's portfolio and STEP projects."
```

Comment:
Can you please rebase this commit onto the previous one that introduces the spurious character?